admin/code/CMSProfileController.php

```diff
@@ -59,6 +57,7 @@ public function canView($member = null) {
 		if(
 			!Permission::checkMember($member, "CMS_ACCESS_LeftAndMain")
```

Comment:
checkMember can take an array of codes to check, as long as the third parameter is left default 'any'